CHANGELOG.md

```diff
@@ -2,7 +2,18 @@
 
 All notable changes to this project will be documented in this file. Dates are displayed in UTC.
 
-Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+
+#### [0.10.1]
+
+> 24 October 2022
+
+- Optional Event Date (End dates shows if different from Start Date)
+- Optional Event Time (Separator show if both date and time are selected)
+- Optional Event Location
+- Simplified Block side menu to reduce the number of collapsible accordions
+- Simplified fields labels
+- Move Heading information to the same location as default variants 
+
```

Comment:
do not modify the CHANGELOG, we handle this auth auto-changelog.  Group your commits with prefixes like: "feat: Optional Event Date", "fix: simplified field labels", etc.